### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by delhomer**
Make the README.md file more easy to read, and add some appealing information (blog and web app URLs), cleaner prediction examples.

Additionally, some cleaning has been done on `images` repo, and article has been removed from the project (it looks better to provide a link to the blog).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,36 @@
...
+associated [web application](http://data.oslandia.io/deeposlandia):
 
-![Example of shape image](./images/shapes_example_2.png)
+![Web application homepage](./images/webapp_homepage.png)
 
 # Content
 
-The project contains some Python materials designed to illustrate the Tensorflow and Keras
-libraries (snippets and notebooks)
+The project contains following folders:
```
Comment (garaud) [inline README.md]: **the** following folders

File: README.md [hunk 2]
```diff
@@ -1,69 +1,36 @@
...
-  on [Oslandia blog](http://oslandia.com/en/blog/) on this topic
-+ [deeposlandia](./deeposlandia) contains main Python modules to train and test convolutional
-  neural networks
-+ [images](./images) contains some example images to illustrate the Mapillary dataset as well as
-  some preprocessing analysis results
-+ [notebooks](./notebooks) contains some Jupyter notebooks that aim at describing data or basic
-  neural network construction
-+ [tests](./tests) contains some test modules to guarantee the functioning of a bunch of snippets;
-  it uses the `pytest` framework.
++ [deeposlandia](./deeposlandia) contains main Python modules to train and test
```
Comment (garaud) [inline README.md]: contains the main Python modules

File: README.md [hunk 3]
```diff
@@ -1,69 +1,36 @@
...
+  convolutional neural networks
++ [examples](./examples) contains some Jupyter notebooks that aim at
+  describing data or basic neural network construction
```
Comment (garaud) [inline README.md]: i should have writtent "... or making a basic neural network"

File: README.md [hunk 4]
```diff
@@ -1,69 +1,36 @@
...
-+ logging
-+ matplotlib
-+ numpy
-+ pandas
-+ PIL
-+ tensorflow
-
-As a remark, the code has been run with Python3 (version 3.5). These dependencies are recalled in
-`setup.py` file, and additional dependencies for developing purpose are listed in
+The code has been run with Python3. The dependencies are recalled in `setup.py`
```
Comment (garaud) [inline README.md]: Python 3 instead of Python3

Comment (garaud) [reply README.md]: If prefer "All dependencies are specified in the `setup.py` file"...

File: README.md [hunk 5]
```diff
@@ -1,69 +1,36 @@
...
-  neural network construction
-+ [tests](./tests) contains some test modules to guarantee the functioning of a bunch of snippets;
-  it uses the `pytest` framework.
++ [deeposlandia](./deeposlandia) contains main Python modules to train and test
+  convolutional neural networks
++ [examples](./examples) contains some Jupyter notebooks that aim at
+  describing data or basic neural network construction
++ [images](./images) contains some example images to illustrate the Mapillary
+  dataset as well as some preprocessing analysis results
++ [tests](./tests) contains some test modules to guarantee the functioning of a
```
Comment (garaud) [inline README.md]: I think it's not necessary to specify that the "tests" folder contains test modules. This is just a suggestion:

[pytest](https://docs.pytest.org/en/latest/) is used to launch several tests from the tests folder.

File: README.md [hunk 6]
```diff
@@ -73,134 +40,79 @@ $ git clone https://github.com/Oslandia/deeposlandia
...
+and 5000 images in the testing set. The testing set is proposed only for a
+model test purpose, it does not contain filtered versions of images. The raw
+dataset contains 66 labels, splitted into 13 categories. The following figure
+depicts a prediction result over the 13-labelled dataset version.
 
-## Model training
+![Example of image, with labels and predictions](./images/mapillary_prediction_example.png)
 
-Then the model training itself may be undertaken:
+## AerialImage (INRIA)
```
Comment (garaud) [inline README.md]: INRIA => Inria

File: deeposlandia/preprocessing.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+Additionally, the preprocessed dataset may contain less images than the raw
+dataset: the `-t`, `-v` and `-T` arguments refer respectively to training,
+validation and testing image quantities. The amount indicated as an example
+correspond to raw dataset size.
+
+For AerialImage dataset, a limited set of image sizes are supported. As smaller
+tiles will be generated by cutting the big original image, a divisor of 5000 is
+expected.
+
+In the Shapes datase case, this preprocessing step generates a bunch of images
```
Comment (garaud) [inline deeposlandia/preprocessing.md]: In the Shape dataset case

**Review by garaud — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (delhomer) [reply README.md]: Modified!

File: README.md (hunk 2 above)
Comment (delhomer) [reply README.md]: Modified!

File: README.md (hunk 3 above)
Comment (delhomer) [reply README.md]: I modified in "that aim at describing data and building basic neural networks"

File: README.md (hunk 4 above)
Comment (delhomer) [reply README.md]: Modified!

File: README.md (hunk 5 above)
Comment (delhomer) [reply README.md]: Modified!

File: README.md (hunk 6 above)
Comment (delhomer) [reply README.md]: Modified!

File: deeposlandia/preprocessing.md (hunk 7 above)
Comment (delhomer) [reply deeposlandia/preprocessing.md]: I removed the capital letter from `Shape` as well.

**Pull request merged**